portal/src/graphql/portal/ForgotPasswordSettings.tsx

```diff
@@ -218,6 +224,14 @@ const ForgotPasswordSettings: React.FC<ForgotPasswordSettingsProps> = function F
     [rawAppConfig, initialState, state, updateAppConfig, updateTemplates]
   );
 
+  // Set state to initialState after updating
+  useEffect(() => {
+    if (!updatingAppConfig || !updatingTemplates) {
+      setState(initialState);
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [updatingAppConfig, updatingTemplates]);
```

Comment:
Can we try to set `key` on `<ForgotPasswordSettings>`? The value of the key `version` should be a increasing integer. Every time a update is done, `version` should be incremented by one. By changing the key, the component will unmount and mount again.